translations/addarr.en-us.yml

```
@@ -1,70 +1,75 @@
-  Select from which season: From which season?
-  Selected and future seasons: Selected and future seasons
-  Season: Season
-  Select all seasons: Select all seasons
-  Deselect all seasons: Deselect all seasons
-  Authorize: You first need to authorize this chat. What is the password?
-  Wrong password: You entered the wrong password. Try again...
-  Chatid added: "This chat is successfully added to the list of allowed chats. Now you can start using this bot. \nIf you entered a command, you need to execute it again."
-  NotAdmin: "You need to be an admin to execute this command."
-  Chatid already allowed: "This chat is already authorized."
-  Help: "Want to see this text again? Use /%{help}.
-    \n• To authenticate before first use you should use /%{authenticate}.
-    \n• To add a movie or series use /%{add}.
-    \n• To delete a movie or series use /%{delete}.
-    \n• To immediately add a movie or series you can use /%{movie} or /%{serie}.
-    \n• To check which series are in Sonarr you can use /%{allSeries}.
-    \n• To check which movies are in Radarr you can use /%{allMovies}.
-    \n• To toggle the up- and download speed of Transmission use /%{transmission}.
-    \n• To change the up- and download speed of Sabnzbd use /%{sabnzbd}."
-  Missing config: "You are missing the following configkeys in your config.yaml-file: %{missingKeys}.
-    \nAdd these before restarting the bot."
-  Wrong values: "The following configkeys has a wrong value in your config.yaml-file: %{wrongValues}.
-    \nChange these before restarting the bot."
+  General: 
+    Start: Start
+    Stop: Stop
+    Find: Find
+    Movie: Movie
+    MovieWithArticle: The movie
+    Series: Series
+    SeriesWithArticle: The series
+    Title: Enter the title
+    WhatIsThis: Is this a movie or a series?
+    New: New search
+    Season: Season
+    End: The current operation terminated by user.
+    PreviousResult: Previous
+    NextResult: Next
+    SelectAnInstance: Please select an instance
+    SelectAPath: Please select a path for the movie or series
+    SelectAQuality: Please select a quality profile for the movie or series
+    SelectFromWhichSeason: From which season?
 
-  messages:
-    Add: Yes, add this %{subject}
-    This: Is this %{subjectWithArticle} you want to add?
-    ThisDelete: Is this %{subjectWithArticle} you want to delete?
+  Actions:
+    Add: Yes, add this
+    Delete: Yes, delete this
+    StopDelete: No, I don't want to delete this
+    Select: Yes, select this
+    SelectedAndFutureSeasons: Selected and future seasons
+    Select all seasons: Select all seasons
+    DeselectAllSeasons: Deselect all seasons
+
+  Authorization:
+    Authorize: You first need to authorize this chat. What is the password?
+    WrongPassword: You entered the wrong password. Try again...
+    ChatID_Added: This chat is successfully added to the list of allowed chats. Now you can start using this bot. \nIf you entered a command, you need to execute it again.

```

To correctly show the enwline (`\n`) you should put this string between `"..."`. Please also check the other translation files

```suggestion
    ChatID_Added: "This chat is successfully added to the list of allowed chats. Now you can start using this bot. \nIf you entered a command, you need to execute it again."
```